docs/registerSwitch_addPorts.commands

```diff
@@ -0,0 +1,30 @@
+Commands to register a switch with HaaS
```

Comment:
We should move these examples to somewhere easier to find. Perhaps place it in `docs/USING.rst` or `examples/curl_commands` ?